hw/syn/tools/dc/parse-syn-report.py

```diff
@@ -68,7 +68,7 @@ def _extract_messages(full_file, results, key, args):
         results['messages'][severity] += re.findall(pattern,
                                                     full_file,
                                                     flags=re.MULTILINE)
-    return results
+    return
```

Comment:
Since this is the end of the function, I think the normal Python style is to omit the return if there's no value. (This applies to quite a few other cases below)

Reply:
oh you're right! will remove